### PR TITLE
docs: document `glimpse()` S3 method for `dm`

### LINKED ISSUE #1002: Document `glimpse.dm()` and possibly other methods
Doesn't necessarily need to be mentioned in a vignette, but at least a help page would be good.

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #1002

### CODE REVIEW

File: R/dm.R [hunk 1]
```diff
@@ -701,6 +701,27 @@ as.list.zoomed_dm <- function(x, ...) {
   as.list(tbl_zoomed(x))
 }
 
+#' Get a glimpse of `dm` object
```
Comment (TSchiefer) [inline R/dm.R:704]:
```suggestion
#' Get a glimpse of your `dm` object
```
Alternatively "of a `dm` object"?

**Review by TSchiefer — APPROVED** [review]
Very nice, thanks!

**Comment by IndrajeetPatil** [comment]
@TSchiefer Thanks, made the change!

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**